lsa-mis/room_ready
# Lra 906 room ready rover interface create navigation for rovers to pick a room

Navigation for Rovers (In Progress)

Features:

- #zones
  - list of all zones
  - option to go to buildings list
- #buildings 
  - list of buildings for specific zone
  - option to choose floor
- #rooms              
  - list of rooms for specific floor
  - option to go to rover form (empty link for now)
- search bar for buildings and rooms
- breadcrumbs for nav


To test:

1. **you need to be a rover**
2. on the navbar, click Rover Form > Choose Room
3. navigate from there
  a) zone > building > room
  b) use breadcrumbs to go back

Questions:

1. Is the page structure fine? (zone > building > room)
2. Any other suggestions for viewing the objects (buildings, rooms) instead of the cards I used? or are they fine?
3. Should I hotwire the search functionality (filters as you type)?
